src/main/java/com/commercetools/paymenttoorderprocessor/PaymentToOrderApplication.java

```diff
@@ -1,32 +1,34 @@
 package com.commercetools.paymenttoorderprocessor;
 
-import java.util.concurrent.TimeUnit;
-
+import io.sphere.sdk.client.BlockingSphereClient;
+import io.sphere.sdk.client.SphereAccessTokenSupplier;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientFactory;
+import io.sphere.sdk.http.HttpClient;
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.DependsOn;
 
-import io.sphere.sdk.client.BlockingSphereClient;
-import io.sphere.sdk.client.SphereAccessTokenSupplier;
-import io.sphere.sdk.client.SphereClient;
-import io.sphere.sdk.client.SphereClientFactory;
-import io.sphere.sdk.http.HttpClient;
+import java.util.concurrent.TimeUnit;
```

Comment:
In the future would be better to put formatting changes into separate PR / commit